plugins/woocommerce/includes/cli/class-wc-cli-com-command.php

```diff
@@ -152,20 +154,19 @@ public static function connect( array $args, array $assoc_args ) {
 			if ( $force ) {
 				WC_Helper::disconnect();
 			} else {
-				WP_CLI::error( 'Your store is already connected.' );
+				WP_CLI::error( __( 'Your store is already connected.', 'woocommerce' ) );
 
 				return;
 			}
 		}
 
-		// phpcs:ignore
-		// @todo add URL to application password section
 		if ( empty( $password ) ) {
-			$password = self::ask( 'Connection password:' );
+			WP_CLI::log( sprintf( __( 'If you don\'t have an application password (not your account password), generate a password from %s', 'woocommerce' ), esc_url( self::APPLICATION_PASSWORD_SECTION_URL ) ) );
```

Comment:
Missing a `// translators:` comment.

```suggestion
			// translators: %s is a URL.
			WP_CLI::log( sprintf( __( 'If you don\'t have an application password (not your account password), generate a password from %s', 'woocommerce' ), esc_url( self::APPLICATION_PASSWORD_SECTION_URL ) ) );
```